spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/support/SpringMvcContract.java

```diff
@@ -45,9 +44,9 @@
 	@Override
 	protected void processAnnotationOnMethod(MethodMetadata data,
 			Annotation methodAnnotation, Method method) {
-		if (!(methodAnnotation instanceof RequestMapping)) {
+		if(!(methodAnnotation instanceof RequestMapping))
```

Comment:
Sorry, I added the same line on my local copy (b/c it was missing when I checked out the code).